wire-runtime/src/test/java/com/squareup/wire/ProtoAdapterTest.java

```diff
@@ -35,6 +35,18 @@
     assertThat(personAdapter.decode(encoded.toByteArray())).isEqualTo(person);
   }
 
+  @Test public void getFromInstanceSameAsFromClass() throws Exception {
+    Person person = new Person.Builder()
+        .id(99)
+        .name("Omar Little")
+        .build();
+
+    ProtoAdapter<Person> instanceAdapter = ProtoAdapter.get(person);
+    ProtoAdapter<Person> classAdapter = ProtoAdapter.get(Person.class);
+
+    assertThat(instanceAdapter).isSameAs(classAdapter);
```

Comment:
FYI you can just call `Person.ADAPTER ` which is the source of truth as the adapter. Calling `get` just does reflection on this field.